File: android-transformer-sample/src/main/java/com/mobandme/sample/app/model/PresentationHome.java
package com.mobandme.sample.app.model;

import com.mobandme.android.transformer.annotations.Mappable;
import com.mobandme.android.transformer.annotations.Mapped;

/**
 * Created by Dimorinny on 20.05.16.
 */
@Mappable(with = NetworkHome.class)
public class PresentationHome {
    @Mapped public String name;
    @Mapped public String city;
}

File: android-transformer-sample-kotlin/src/main/java/com/mobandme/sample/kotlin/app/model/PresentationHome.kt
package com.mobandme.sample.kotlin.app.model

import com.mobandme.android.transformer.annotations.Mappable
import com.mobandme.android.transformer.annotations.Mapped

/**
 * Created by Dimorinny on 20.05.16.
 */
@Mappable(with = NetworkHome::class)
data class PresentationHome(
        @Mapped
        var name: String = "",

        @Mapped
        var city: String = ""
)